package.json

```diff
@@ -95,7 +95,7 @@
   "devDependencies": {
     "@next/eslint-plugin-next": "^12.0.10",
     "@testing-library/jest-dom": "^5.16.2",
-    "@testing-library/react": "^12.1.3",
+    "@testing-library/react": "^11.2.5",
```

Comment:
Instead of downgrade this, maybe try upgrade uikit one

Reply:
Done